setup.py

```diff
@@ -54,9 +54,9 @@
         ],
         'tinybase': [
             'distributed==2023.12.1',
-            'pympipool==0.7.9'
+            'pympipool==0.7.9',
```

Comment:
@pmrv The `setuptools` issue was a missing `,` after `pympipool`. Then the two strings `'pympipool==0.7.9'` and `'h5io_browser==0.0.4',` are combined to once string `'pympipool==0.7.9h5io_browser==0.0.4'` which does not follow the requirements for the `setup.py` file. 
```
error in pyiron_contrib setup command: 'extras_require' must be a dictionary whose values are strings or lists of strings containing valid project/version requirement specifiers.
```

Reply:
Ah, yikes.  Thanks for catching that!